src/fsharp/FSharp.Compiler.Interactive.Settings/FSharp.Compiler.Interactive.Settings.fsproj

```diff
@@ -5,7 +5,6 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netstandard2.0</TargetFrameworks>
```

Comment:
Doesn't this cause it to fail on mac/linux?

Reply:
it was a mistake, I need to revert it.

Reply:
Although no, it doesn't and I don't really understand why.

Reply:
@brettfo, you can build net472 on linux.  You can't run it, but you can for sure build it.